### PR TITLE
LPS-137647 Translations are missing for some DM shortcut dropdown selections

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-137647

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:test:relevant

File: modules/apps/document-library-google-docs/document-library-google-docs/src/main/java/com/liferay/document/library/google/docs/internal/util/GoogleDocsDLFileEntryTypeHelper.java [hunk 1]
```diff
@@ -125,11 +124,48 @@ private void _addGoogleDocsDLFileEntryType(long ddmStructureId)
...
-			GoogleDocsConstants.DL_FILE_ENTRY_TYPE_KEY, nameMap, descriptionMap,
+			GoogleDocsConstants.DL_FILE_ENTRY_TYPE_KEY,
+			_externalGoogleDriveShortcutNameMap(
+				LanguageUtil.getAvailableLocales()),
+			descriptionMap,
 			DLFileEntryTypeConstants.FILE_ENTRY_TYPE_SCOPE_SYSTEM,
 			serviceContext);
 	}
 
+	private Map<Locale, String> _externalGoogleDriveShortcutNameMap(
```
Comment (adolfopa) [inline modules/apps/document-library-google-docs/document-library-google-docs/src/main/java/com/liferay/document/library/google/docs/internal/util/GoogleDocsDLFileEntryTypeHelper.java:135]: Rename this to _getExternalGoogleDriveShortcutNameMap

File: modules/apps/document-library-google-docs/document-library-google-docs/src/main/java/com/liferay/document/library/google/docs/internal/util/GoogleDocsDLFileEntryTypeHelper.java [hunk 2]
```diff
@@ -69,6 +70,8 @@ public void addGoogleDocsDLFileEntryType() throws Exception {
 		if (dlFileEntryType == null) {
 			_addGoogleDocsDLFileEntryType(ddmStructure.getStructureId());
 		}
+
```
Comment (adolfopa) [inline modules/apps/document-library-google-docs/document-library-google-docs/src/main/java/com/liferay/document/library/google/docs/internal/util/GoogleDocsDLFileEntryTypeHelper.java:73]: Shouldn't we add the update call in an else clause? If dlFileEntryType == null, the name map will be correctly initialized on add; we would only need to update if it was not null.

File: modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/helper/DLVideoExternalShortcutDLFileEntryTypeHelper.java [hunk 3]
```diff
@@ -73,6 +72,8 @@ public void addDLVideoExternalShortcutDLFileEntryType() throws Exception {
 			_addDLVideoExternalShortcutDLFileEntryType(
 				ddmStructure.getStructureId());
 		}
+
+		_updateDLFileEntryTypeNameMap(dlFileEntryType);
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/helper/DLVideoExternalShortcutDLFileEntryTypeHelper.java:76]: Same things I commented for the Google shortcut apply here: method rename and else clause.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**